### ISSUE TITLE
UI: copy button doesn't do anything when Deposit wallet address is displayed

### ISSUE THREAD

**Issue by norwnd** [decred/dcrdex#1906]
Hi, noticed this while poking at DEX (on latest master), maybe it was mentioned somewhere already, I couldn't find anything related in issues/pulls though,

when I'm clicking on Receive button to open Deposit address window (as is shown on screenshot below) there is a button next to the address string that I can click on, but when clicked it doesn't seem to do anything (it doesn't get copied to clipboard),
this looks like a bug to me, or at least misleading behavior.

It's reproducible with any wallet (BTC, DCR) and in all browsers I tried.

<img width="421" alt="image" src="https://user-images.githubusercontent.com/112318969/196058780-d01c85c7-d50c-416b-bfce-7b5714e9dde3.png">

**Comment by norwnd** [comment]
As an afterthought, maybe it is possible to fully avoid this 2nd click by:
- saving address in the clip-board **when Receive button is clicked** (as on screenshot below)
- remove this copy-button that's sitting next to address and instead **show some animation** that means "copied successfully"

or something along these lines

**Comment by norwnd** [comment]
This same copy button is working in Decrediton, by the way

<img width="368" alt="image" src="https://user-images.githubusercontent.com/112318969/196059359-cc58fb46-54c2-4978-ad0b-89b3b90d45a9.png">

**Comment by chappjc** [comment]
Unable to reproduce with Firefox or Chromium.

What browser?

Will you verify that you built both the site assets and *then* build dexc?  https://github.com/decred/dcrdex/wiki/Client-Installation-and-Configuration#build-from-source

**Comment by norwnd** [comment]
> What browser?

I tried Chrome/Brave/Firefox. 

I've re-run instructions in **Build from Source** section, and now the issue disappeared,

I haven't seen **Build from Source** section in the docs before, so I kinda tried to figure out on my own how to build it, and what I did was (based on some scripts in this repo):
```
pushd client/webserver/site
npm ci
npm run build
popd
(cd client/cmd/dexc/ && CGO_ENABLED=0 GO111MODULE=on go build)
```
I already tried to build DEX client before (don't remember successfully or not), and now I got back to it and didn't do `npm clean-install` when re-building it again. So I assume this might be the reason for what I was observing with this copy-button.

I think this issue can be closed, thanks for pointing me to the relevant docs section.

And I think UX could be slightly improved if there was no need for that 2nd click after all - https://github.com/decred/dcrdex/issues/1906#issuecomment-1280061200
